packages/x-data-grid-generator/package.json

```diff
@@ -35,6 +35,8 @@
   "dependencies": {
     "@babel/runtime": "^7.26.10",
     "@mui/x-data-grid-premium": "workspace:*",
+    "@mui/x-data-grid-pro": "workspace:*",
+    "@mui/x-data-grid": "workspace:*",
```

Comment:
Doesn't this chain of dependencies resolve automatically?

Reply:
The generator package is directly importing from community or pro versions in some cases.